### ISSUE TITLE
Ready-to-use Checkpoint of this model

### ISSUE THREAD

**Issue by DaehanKim** [PlusLabNLP/story-gen-BART#7]
Dear Authors,
I wonder whether you can provide ready-to-use checkpoint of this model to use for `inference_plot.py` and `inference_story.py`.
I only see instructions on pretrained BART and Tokenizers checkpoint. If I'm missing something, please let me know.
Thank you.

**Comment by figlang2022sharedtask** [comment]
https://drive.google.com/drive/folders/1cOouBxVsORnNdQJuZlH9fu3ACc7p9CwG?usp=sharing

I think this folder has everything you need

**Comment by tuhinjubcse** [comment]
I dont think the event discrimators are here , but can you please train them , sorry i dont have them with me i the drive @seraphinatarrant can help more

**Comment by DaehanKim** [comment]
Thank you for the reply.
Where can I find a script for coreference resolution and semantic role labeling? If there's no such script, how can I process raw story txt into silver standard plots as in the paper? 
Plus, do we use the same plot model for revising plots too?
Thanks in advance

**Comment by tuhinjubcse** [comment]
https://github.com/PlusLabNLP/story-gen-BART/tree/master/srl_plot_preprocessing

**Comment by DaehanKim** [comment]
Thank you!

**Comment by DaehanKim** [comment]
![image](https://user-images.githubusercontent.com/20675681/185551833-a91130b9-192d-4751-ac50-64c15a9ef876.png)

What does this Sharp mean in the literature and implementation? And I wonder how WrintingPrompts datasets were preprocessed before coref & srl. my guess is
- removed [WP] things in source (title)
- replace <newline> into </s>

Is that all? I'm asking this because original data is not available here (I need access but the intern seems offline).
![image](https://user-images.githubusercontent.com/20675681/185552319-21a5e1a4-ff9d-4b03-be32-c00afa7dbdc6.png)

**Comment by DaehanKim** [comment]
How did you split WritingPrompts' train/valid/test? Since the original data is already splitted and I can find only [train/valid].[source/target] in your drive.. In the appendix, you said you splitted it into 65/10/10 for story generator and 15 for training rescorer.

**Comment by DaehanKim** [comment]
Hi, I'm trying to generate plot using two rescorers(entity and relevance). 
It requires `input0.dict.txt` and `label.dict.txt` in roberta checkpoint folder but I can't find these files at your drive link. Should I retrain all these models?
